lucille-core/src/test/java/com/kmwllc/lucille/stage/ParseFloatsTest.java

```diff
@@ -0,0 +1,66 @@
+package com.kmwllc.lucille.stage;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+import java.util.List;
+import org.junit.Test;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+
+public class ParseFloatsTest {
+
+  private StageFactory factory = StageFactory.of(ParseFloats.class);
+
+  @Test
+  public void testMalformedConfig() {
+    assertThrows(StageException.class, () -> factory.get("ParseFloatsTest/bad.conf"));
+  }
+
+  @Test
+  public void testProcessDocument() throws StageException {
+    Stage stage = factory.get("ParseFloatsTest/basic.conf");
+
+    Document bad = Document.create("doc1");
+    Document empty = Document.create("doc2");
+    Document simple = Document.create("doc3");
+    Document noField = Document.create("doc4");
+    Document floats = Document.create("doc5");
+    Document badList = Document.create("doc6");
+
+    bad.setOrAdd("source", "not a float");
+    empty.setOrAdd("source", "[]");
+    simple.setOrAdd("source", "[1, 2, 3]");
+    floats.setOrAdd("source", "[1.2, 2.5, 3.1]");
+    badList.setOrAdd("source", "[1, a, 3]");
```

Comment:
suggest putting the a in quotes so the test string is actually valid json
the "bad" case already covers invalid json